app/views/layouts/application.html.erb

```diff
@@ -32,16 +32,17 @@
     <%- end %>  
     <%= preload_script 'browser-detect' %>
 
-    <%= preload_script "locales/#{I18n.locale}" %>
-    <%- if ExtraLocalesController.client_overrides_exist? %>
-      <%= preload_script_url ExtraLocalesController.url('overrides') %>
-    <%- end %>
     <%= preload_script "vendor" %>
     <%= preload_script "discourse" %>
     <%- Discourse.find_plugin_js_assets(include_official: allow_plugins?, include_unofficial: allow_third_party_plugins?, request: request).each do |file| %>
       <%= preload_script file %>
     <%- end %>
 
+    <%= preload_script "locales/#{I18n.locale}" %>
+    <%- if ExtraLocalesController.client_overrides_exist? %>
+      <%= preload_script_url ExtraLocalesController.url('overrides') %>
+    <%- end %>
+
     <%- if staff? %>
```

Comment:
Unrelated to this PR, but it's Interesting that admin.js/wizard.js actually appears after the app bundle, whereas in the bootrsap-json version they appear before. We should probably update the bootstrap-json version to match this. Something happened there to cause the vendor/app bundle to be pushed into the beginning of the body tag when in `app/index.html` they appear in a spot resembling what's in here.

Reply:
(Ultimately, it would be even better to ensure things aren't as ordering-dependent, but to the extent that things are sensitive to that today it seems good to make sure things match up)

Reply:
Yup, makes sense. @CvX was looking at the stuff being pushed into `<body>` by embroider - it seems that we're using non-compliant custom HTML tags in the `<head>`, which Embroider/webpack rejects and pushes into the body.

Ideally we need to replace the custom `<discourse-chunked-script>` tags with something else. Probably just a specially-formatted comment before/after 👀

Reply:
Ah I see, it must be that jsdom has pretty rigid (and correct) rules about what can go inside `<head>` and the error correction kicked in